x-pack/test/security_solution_cypress/cypress/e2e/detection_response/detection_engine/detection_alerts/assignments/assignments.cy.ts

```diff
@@ -155,71 +126,57 @@ describe('Alert user assignment - ESS & Serverless', { tags: ['@ess', '@serverle
 
     it('updating assignees via `More actions` in alerts table', () => {
       // Initially assigned users
-      const initialAssignees = [ROLES.detections_admin, ROLES.t1_analyst];
+      const initialAssignees = [getDefaultUserName()];
       updateAssigneesForFirstAlert(initialAssignees);
       alertsTableShowsAssigneesForAlert(initialAssignees);
 
       // Update assignees
-      const updatedAssignees = [ROLES.t1_analyst, ROLES.t2_analyst];
+      const updatedAssignees = [getDefaultUserName()];
```

Comment:
I don't know if these "update assignees" tests still make sense: both the initial assignees and the updated assignees are the same, so these are all prone to false positives.